subprojects/docs/src/snippets/dependencyManagement/declaringDependencies-fileDependencies/tests/file-dependencies.sample.conf

```diff
@@ -1,2 +1,2 @@
 executable: gradle
-args: tasks
+args: copyLibs
```

Comment:
I couldn't gather the reason for this and other similar changes elsewhere. Could you clarify, Mike?